clients/python/girder_client/__main__.py

```diff
@@ -17,7 +17,9 @@
 #  limitations under the License.
 ###############################################################################
 
-from girder_client import cli  # pragma: no cover
+import warnings
+from girder_client import cli
 
-
-cli.main()  # pragma: no cover
+if __name__ == '__main__':
+    warnings.warn('Deprecation notice: Use "girder-client" to run the CLI.', DeprecationWarning)
```

Comment:
I tried running `python -m girder_client`, and I'm not seeing this warning.  Maybe I have something in my environment disabling it...

Reply:
Fixed to always print.